bitnami/solr/templates/statefulset.yaml

```
@@ -262,7 +262,7 @@ spec:
                 {{- if .Values.auth.enabled }}
                 SOLR_AUTH_TYPE=basic SOLR_AUTHENTICATION_OPTS="-Dbasicauth=${SOLR_ADMIN_USERNAME}:${SOLR_ADMIN_PASSWORD}" \
                 {{- end }}
-                solr assert --timeout {{ printf "%v000" .Values.livenessProbe.timeoutSeconds | quote }} --cloud "{{ ternary "https" "http" .Values.tls.enabled }}://localhost:${SOLR_PORT_NUMBER}/solr/"
+                solr assert --timeout {{ printf "%v000" .Values.livenessProbe.timeoutSeconds | quote }} --{{ ternary "" "not-" .Values.cloudEnabled }}cloud "{{ ternary "https" "http" .Values.tls.enabled }}://localhost:${SOLR_PORT_NUMBER}/solr/"
```

Hello @RichardSufliarsky, thanks for the fix! The change itself looks good but I would add the rest of the flag inside the ternary. It would make it slightly more verbose but I think we'll gain in readability, wdyt?
```suggestion
                solr assert --timeout {{ printf "%v000" .Values.livenessProbe.timeoutSeconds | quote }} {{ ternary "--cloud" "--not-cloud" .Values.cloudEnabled }} "{{ ternary "https" "http" .Values.tls.enabled }}://localhost:${SOLR_PORT_NUMBER}/solr/"
```

Regarding the still appearing issues, have you detected them once the app is running or are they just present at the start?

---

Hi @FraPazGal, you are right. It will be better. It is more readable.
Regarding the lexceptions in the log, they repeat every 10 seconds.
2022-06-08 21:39:22.184 ERROR (qtp1057468716-17) [] o.a.s.h.RequestHandlerBase org.apache.solr.common.SolrException: Solr instance is not running in SolrCloud mode. => org.apache.solr.common.SolrException: Solr instance is not running in SolrCloud mode.
2022-06-08 21:39:22.184 INFO  (qtp1057468716-17) [] o.a.s.s.HttpSolrCall [admin] webapp=null path=/admin/collections params={action=CLUSTERSTATUS&wt=javabin&version=2} status=400 QTime=0
2022-06-08 21:39:22.263 ERROR (qtp1057468716-23) [] o.a.s.h.RequestHandlerBase org.apache.solr.common.SolrException: Solr instance is not running in SolrCloud mode. => org.apache.solr.common.SolrException: Solr instance is not running in SolrCloud mode.
2022-06-08 21:39:22.263 INFO  (qtp1057468716-23) [] o.a.s.s.HttpSolrCall [admin] webapp=null path=/admin/collections params={action=CLUSTERSTATUS&wt=javabin&version=2} status=400 QTime=0
[solr-test.log](https://github.com/bitnami/charts/files/8865218/solr-test.log)